### ISSUE TITLE
Add files from different directories

### ISSUE THREAD

**Issue by Shadester** [gavinbenda/platinum-md#114]
Normally when writing, you would be adding a few files and then start transferring. If you have the files in different directories, it will need to transfer then per directory. It would be great if all the selection of files can be done first and then start the transfer.

**Comment by gavinbenda** [comment]
Good idea - this needs a bit of thought as currently it simply loops over each selected track one by one in a very predictable way. There is another request to support drag-and-drop which might also form part of this change.

I'd probably approach it by having the existing directly window as more of a 'Queue' panel and make a totally new interface to browse/select tracks and add them to the queue.
